lib/components/Input/text-field-with-steppers/index.tsx

```diff
@@ -0,0 +1,90 @@
+import { ComponentProps, forwardRef } from "react";
+import Input from "../base";
+import React from "react";
+import clsx from "clsx";
+import {
+    LabelPairedPlusSmRegularIcon,
+    LabelPairedMinusSmRegularIcon,
+} from "@deriv/quill-icons";
+import "./text-field-with-steppers.scss";
+import { TLeftOrRight } from "@types";
+
+export interface TextFieldWithSteppersProps extends ComponentProps<typeof Input> {
+    unit?: string;
+    decimals?: number;
+    unitPlacement?: TLeftOrRight;
+}
+
+export const TextFieldWithSteppers = forwardRef<HTMLInputElement, TextFieldWithSteppersProps>(
+    (props, ref) => {
+        const { value, onChange, disabled, textAlignment, placeholder = 0, unit = '', decimals = 2, unitPlacement = 'left' } = props;
+
+        const getFormatValue = (value: number) => {
+            let formatValue = value;
+
+            if (typeof formatValue === 'number') {
+                formatValue = parseFloat(formatValue.toFixed(decimals));
+            }
+
+            return formatValue;
```

Comment:
`return typeof formatValue === 'number' ? parseFloat(formatValue.toFixed(decimals)) : formatValue`
May be it will be shorter?

Reply:
Also, you determine the type of `value` as `number` ( `const getFormatValue = (value: number) =>`), do we need to check it again?